compat/src/index.d.ts

```diff
@@ -272,7 +264,7 @@ declare namespace React {
 	}
 
 	export function forwardRef<R, P = {}>(
-		fn: ForwardFn<P, R>
+		fn: ForwardRefRenderFunction<R, P>
```

Comment:
This is from #4675 -- `ForwardFn` is not the name in React & notably it has the generics reversed. It was marked as deprecated in that PR so all this is doing is replacing it with the correct, non-deprecated version (the internals of both types match).